php-checks/src/main/java/org/sonar/php/checks/security/LDAPAuthenticatedConnectionCheck.java

```diff
@@ -52,14 +52,15 @@ public void visitCompilationUnit(CompilationUnitTree tree) {
 
   @Override
   protected void createIssue(FunctionCallTree tree) {
-    if (argumentIsNullOrEmptyString(tree, USERNAME_ARG_INDEX) || argumentIsNullOrEmptyString(tree, PASSWORD_ARG_INDEX)) {
+    if (argumentIsNullOrEmptyString(tree, "bind_rdn", 1) || argumentIsNullOrEmptyString(tree, "bind_password", 2)) {
       context().newIssue(this, tree, MESSAGE);
     }
   }
 
-  private boolean argumentIsNullOrEmptyString(FunctionCallTree tree, int argumentIndex) {
-    if (tree.arguments().size() > argumentIndex) {
-      ExpressionTree valueArgument = getAssignedValue(tree.arguments().get(argumentIndex));
+  private boolean argumentIsNullOrEmptyString(FunctionCallTree tree, String argumentName, int argumentIndex) {
+    Optional<CallArgumentTree> argument = CheckUtils.argument(tree, argumentName, argumentIndex);
+    if (argument.isPresent()) {
+      ExpressionTree valueArgument = getAssignedValue(argument.get().value());
```

Comment:
I would call this `argumentValue` for consistence with the other cases

Reply:
Fixed.